capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx

```diff
@@ -40,12 +41,10 @@ export default function ExperienceInfo({ restaurants }) {
   const { username } = useContext(UserContext);
   const [indexRestaurant, setIndexRestaurant] = useState(0);
   const [activeFeedback, setActiveFeedback] = useState([]);
-  console.log(activeFeedback)
   const [newReview, setNewReview] = useState([]);
   const [isRated, setIsRated] = useState(false);
   // Get the restaurant viewing now
-  const currentRestaurant =
-    restaurants.length > 0 ? restaurants[indexRestaurant] : null;
+  const currentRestaurant = restaurants.length > 0 ? restaurants[indexRestaurant] : null;
```

Comment:
I know you responded to my [earlier comment](https://github.com/carlosvegap/capstone-kickoff/pull/41/files#r931302993), but this still doesn't look right to me. In JS nothing will happen if we index into an invalid array location, but in many languages this would throw an exception.

Are there any issues with updating this?

Reply:
Oh, so far I haven't faced any problems, but I will definitely be on the lookout for any sort of problems there